tests/php/Forms/FieldListTest.php

```diff
@@ -259,6 +259,8 @@ public function testFindTab()
         $this->assertNull($fields->findTab('More'));
         $this->assertEquals($fields->findTab('Root.More'), $more);
         $this->assertEquals($fields->findTab('Root.More.Tab4'), $tab4);
+
```

Comment:
```suggestion
```

Reply:
I'm leaving this in - it matches the way new lines are used above this line in the same method.